README.md

```diff
@@ -42,6 +42,75 @@ console.error(kayvee.formatLog("test_source", kayvee.INFO, "title", {"foo" : 1,
 # {"foo":1,"bar":"baz","source":"test_source","level":"info","title":"title"}
 ```
 
+## Example: Kayvee Log Routing
+
+Log routing is a mechanism for defining where log lines should go once they've entered Clever's logging pipeline.   Routes are defined in a yaml file called kvconfig.yml.  Here's an example of a log routing rule that sends a slack message:
+
+```js
+// main.js
+const kv = require("../kayvee-js");
+kv.setGlobalRouting("./kvconfig.yml");
+
+const log = new kv.logger("myApp");
+
+module.exports = (cb) => {
+    // Simple debugging
+    log.debug("Service has started");
+
+    // Make a query and log its length
+    let query_start = Date.now();
+    log.gauge("QueryTime", Date.now() - query_start);
```

Comment:
This doesn't seem related to sending to slack-- should remove it from the example?